dap-etl/transformation/src/main/scala/org/broadinstitute/monster/dap/common/HealthTransformations.scala

```diff
@@ -105,15 +110,52 @@ object HealthTransformations {
       }
     }
 
-    Iterable.concat(cgs, dxs)
+    // Fill in with dummy health condition records for instances where the top level condition question
+    // answered "yes" but no specific health conditions were filled in
+    val cgsFillIn =
+      createFillInConditionRecords(
+        cgs,
+        rawRecord,
+        isCongenital = true,
+        (h: HealthConditionType) => h.cgKey
+      )
+    val dxsFillIn =
+      createFillInConditionRecords(
+        dxs,
+        rawRecord,
+        isCongenital = false,
+        (h: HealthConditionType) => h.dxKey
+      )
+
+    Iterable.concat(cgs, dxs, cgsFillIn, dxsFillIn)
+  }
+
+  def createFillInConditionRecords(
+    base: Iterable[HlesHealthCondition],
+    rawRecord: RawRecord,
+    isCongenital: Boolean,
+    keyGetter: (HealthConditionType) => Option[HealthConditionKey]
+  ): Iterable[HlesHealthCondition] = {
+    HealthConditionType.values
+      .filterNot(h => base.exists(c => c.hsConditionType == h.value) || keyGetter(h).isEmpty)
+      .filter(h => rawRecord.getBoolean(keyGetter(h).get.categoryGate))
+      .map { hc =>
+        createHealthConditionRow(
+          rawRecord,
+          keyGetter(hc).get.dataPrefix,
```

Comment:
I don't like this `get` here, but we should be protected from an error by the filter above: (`keyGetter(h).isEmpty`)